mathics/builtin/intfns/divlike.py

```diff
@@ -192,6 +241,43 @@ def apply(self, n: Integer, m: Integer, evaluation):
         return Integer(n % m)
 
 
+class ModularInverse(SympyFunction):
+    """
+    <dl>
+      <dt>'ModularInverse[$k$, $n$]'
+      <dd>returns $k$ modulo $n$.
```

Comment:
returns the modular inverse k^(-1) mod n.

Reply:
Ok - good. See 105803c8